routes/api_v1.php

```diff
@@ -1212,7 +1212,16 @@
                         Route::delete('', ['middleware' => 'auth.user', 'uses' => 'OAuth2SummitSponsorApiController@deleteSocialNetwork']);
                     });
                 });
-
+                // extra questions
+                Route::group(['prefix' => 'extra-questions'], function () {
```

Comment:
we need to create metadata endpoint ( check SDS)

 Route::get('metadata', ['middleware' => 'auth.user', 'uses' => 'OAuth2SummitSponsorApiController@ getMetadata']);
like here
https://github.com/OpenStackweb/summit-api/blob/a02088453cff6fb953998a34f15f66a9d107ce17/app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitOrderExtraQuestionTypeApiController.php#L95